docs/bom/create-a-bill-of-materials.md

```diff
@@ -0,0 +1,251 @@
+- [Create your Bill of Materials](#create-your-bill-of-materials)
+  - [Thinking about your Release](#thinking-about-your-release)
+  - [Generating the SBOM](#generating-the-sbom)
+    - [Namespace](#namespace)
+    - [Simplest Use Case: One Package](#simplest-use-case-one-package)
+    - [Adding Additional Sources](#adding-additional-sources)
+    - [Example: Generate an SBOM for etcd](#example-generate-an-sbom-for-etcd)
+<!-- /toc -->
+
+## Introduction
+
+To generate a Bill of Materials for your project, simply download
+`bom`, our utility that leverages the code found in this repo and
+point it to your project:
+
+```
+cd gitrepo/
+bom -n 'http://mybom.com/' . 
+```
+
+All of the tool's options are explained on its page. Keep reading
+for more information about our tools, SBOMs, and the SPDX standard. 
+
+### What is a Bill of Materials?
+
+A _Software Bill of Materials_ (often BOM or SBOM for short) is a
+manifest that lists everything included in a software release.
+"Everything" can take different meanings: software packages or images,
+documentation, tarballs, single files. These pieces can be components
+or source code, variants of the same artifact (eg a binary for different
+platforms).
+
+An SBOM can also provide visibility on the dependencies of your project.
+There are many types of dependencies and many reasons that consumers of
+a project need to know them: security,  compliance, compatibility.
+
+Finally, a Bill of Materials allows software creators to express licensing
+information for their project as a whole, but also for individual pieces
+and it's dependencies. You can release your project under the Apache 2.0
+license but have its documentation published under Creative Commons. Then,
+there are all of your dependencies' original licenses. A well-written
+SBOM can express all of them in the same document. 
+
+### bom: A tool to generate SBOMs for your Project
+
+As part of the effort to produce a bill of materials for Kubernetes, SIG
+Release developed a set of libraries to produce fully compliant SPDX SBOMs.
+Our tools support license scanning, image layer analyzers, processing of
+golang dependencies, and other features. These libraries are available for
+other projects to automate the production of their own Bills of Materials.
+
+For simpler use cases, all of our SBOM automation is also available in
+a general-purpose tool called `bom`. You can find all the options that
+`bom` supports in its README.md.
+
+`bom` supports generating Bills of Materials in SPDX compliant tag-value
+format. It can process directories, single files, read the contents of
+container images (both from docker tar archives and directly from registries),
+and tarred sources.
+
+In addition `bom` will scan your code to find licensing information. Its
+classifier supports detection of all the SPDX recognized licenses.
+
+## SPDX: Software Package Data Exchange
+
+SPDX or _Software Package Data Exchange_ is an open standard to create
```

Comment:
Let's add a link to an official external resource for further readings.

Reply:
Done, added a links to important terms in the spec